projects.json

```diff
@@ -2878,7 +2878,7 @@
     "repository": "Git",
     "url": "https://github.com/apple/swift-nio",
     "path": "swift-nio",
-    "branch": "master",
+    "branch": "main",
```

Comment:
@shahmishal how does the interaction between the branch and the commit hashes work? should this be based on a tag?

Reply:
All of the CI infrastructure use hash, not the branch info. We added this option to be able to know development branch for the project, normally it's main/master but sometime people use different branch name.